### PR TITLE
changed around an if statement

### PR DESCRIPTION
**Pull request by J3fftw1**
changed around an if statement because you return false on the else before aswell so this makes it overall the same and OCD

### CODE REVIEW

File: src/main/java/io/github/seggan/slimypowersuits/SuitUtils.java
Comment (TheBusyBiscuit) [inline src/main/java/io/github/seggan/slimypowersuits/SuitUtils.java]: This is completely redundant.

An instanceof check is already an extension of a null check since null isn't an instance of anything.

```suggestion
        return item instanceof SuitPiece;
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Comment by TheBusyBiscuit** [comment]
@Seggan Also anyone on GitHub can review code :P
It has nothing to do with who has push access. A review is just a fancy form of comment with line markings after all.

**Review by Seggan — APPROVED** [review]
Looks good. Will merge when build 8 is ready.

**Pull request merged**
